### DISCUSSION TITLE
Git is broken

### DISCUSSION THREAD

**Discussion by RoryNesbitt** [requarks/wiki#6272] [Error / Bug Report] [answered]
I have two instances of wiki.js running. both in docker on unraid

I noticed today when saving things I was getting this error:
```
Error: spawn git ENOENT at Process.ChildProcess._handle.onexit (node:internal/child_process:285:19) at onErrorNT (node:internal/child_process:485:16) at processTicksAndRejections (node:internal/process/task_queues:83:21)
```
After which the page would say there was a conflict but when I selected local it was still saving.

I have then gone in to the storage section and my git wasn't syncing, the error given was the same as this one.
I then checked my second instance which was fine (last updated a week ago), but when I clicked force sync it gave the same error.
Finally I manually updated github and then deleted all the wiki.js files and stared from scratch with the intent of pulling everything back from github. But even on a completely fresh install the git storage page gives this error

**Comment by fumbles** [comment]
Are they HA pods? both using the same database?

**Reply by RoryNesbitt** [reply]
No they're both completely seperate from each other. Same server but different wikis

**Comment by RoryNesbitt** [comment, marked as answer]
This seems to have been an issue with the container [[cite](https://github.com/linuxserver/docker-wikijs/issues/24)] and is now resolved
